SU-SWS/stanford_magazine
# Redesign cjw

# READY FOR REVIEW

# Summary
This:
- Adds help text for the issue number
- Removes the layout taxonomy from the feature
- Adds an update hook to remove the `field_s_mag_issue_layout` field

# Needed By (Date)
ASAPP

# Urgency

# Steps to Test

1. Pull this branch onto a jse-soe-new site
2. Revert stanford_magazine_issue: `drush fr stanford_magazine_issue -y --force`
2. Run `drush updb`
3.  Verify the `field_s_mag_issue_layout` field no longer exists
5. Verify the help text for the issue number exists

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOE-1890



# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
